.docs/content/1.installation/2.windows/2.wsl2-with-genie.md

```diff
@@ -0,0 +1,100 @@
+# WSL2 with genie
+
+::alert{type="warning"}
+This installation method is not recommended for production environments.
+::
+
+This installation will use [`genie`](https://github.com/arkane-systems/genie). Genie is a tool which gives you a way to run systemd as pid 1. The usage of this tool will disable some Windows goodies within your Ubuntu distribution.
+
+::alert{type="warning"}
+We does not recommend to use this installation method, even for development environment since WSL2 support natively `systemd`. We recommend to use [WSL2 with systemd](./1.wsl2-with-systemd.md).
```

Comment:
```suggestion
We do not recommend to use this installation method, even for development environment since WSL2 support natively `systemd`. We recommend to use [WSL2 with systemd](./1.wsl2-with-systemd.md).
```